solr/core/src/java/org/apache/solr/core/SolrCore.java

```diff
@@ -3032,6 +3034,7 @@ public PluginBag<QueryResponseWriter> getResponseWriters() {
     m.put("csv", new CSVResponseWriter());
     m.put("schema.xml", new SchemaXmlResponseWriter());
     m.put("smile", new SmileResponseWriter());
+    m.put(PROMETHEUS_METRICS_WT, new PrometheusResponseWriter());
```

Comment:
This is so special-purpose that I think it doesn't belong here.  Can we get away with not registering it?

Reply:
Could not figure out a way to do with without registering it. Is there some example somewhere where a response writer exists without registering the class?

Reply:
Nevermind.  We even have schema xml format, another special purpose one.  Maybe some day this could be improved.

Reply:
I filed https://issues.apache.org/jira/browse/SOLR-17354 for improving this matter.